### ISSUE TITLE
Fresh Install of Development commit - Doesn't auto-delete install/ folder after installation completed

### ISSUE THREAD

**Issue by wirerydr** [nilsteampassnet/TeamPass#1881]
### Steps to reproduce
1. Complete a fesh installation of Teampass (tested with latest development commit 716c1b8).
2. Once at the _**Resume**_ screen, verify that the _**install/**_ folder still exists.
3. Click _"Move to home page"_.

### Expected behaviour
The main login screen for TeamPass should be displayed, allowing the user to log in.

### Actual behaviour
A blank screen is displayed, and the following error is observed in the Apache logs (behaviour repeats if the page is refreshed):
```
[Sat Aug 26 14:26:44.991392 2017] [php7:notice] [pid 4908] [client 192.168.240.120:57183] PHP Fatal error:  Uncaught Error: Call to undefined function chmodRecursive() in /opt/teampass/teampass__20170822-test716c1b8/sources/core.php:116\nStack trace:\n#0 /opt/teampass/te
ampass__20170822-test716c1b8/index.php(92): require_once()\n#1 {main}\n  thrown in /opt/teampass/teampass__20170822-test716c1b8/sources/core.php on line 116, referer: https://www.example.com/tptest/install/install.php
```
Once the _**install/**_ is manually deleted/moved/renamed and the webpage is refreshed, the Login page gets displayed.

### Server configuration
**Operating system**: _Fedora 26 (x86_64)_

**Web server:** _Apache 2.4.27_

**Database:** _MariaDB 10.1.25_

**PHP version:** _PHP 7.1.8_

**Teampass version:** _latest development commit 716c1b8_

**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**  _Fresh install of commit 716c1b8_

#### Web server error log
```
[Sat Aug 26 14:26:44.991392 2017] [php7:notice] [pid 4908] [client 192.168.240.120:57183] PHP Fatal error:  Uncaught Error: Call to undefined function chmodRecursive() in /opt/teampass/teampass__20170822-test716c1b8/sources/core.php:116\nStack trace:\n#0 /opt/teampass/te
ampass__20170822-test716c1b8/index.php(92): require_once()\n#1 {main}\n  thrown in /opt/teampass/teampass__20170822-test716c1b8/sources/core.php on line 116, referer: https://www.example.com/tptest/install/install.php
```

### Additional Diagnostics
```
# grep -nR chmodRecursive
install/install.queries.php:20:function chmodRecursive($dir, $dirPermissions, $filePermissions)
install/install.queries.php:33:                $res = @chmodRecursive($fullPath, $dirPermissions, $filePermissions);
install/install.queries.php:1116:                        $result = chmodRecursive($session_abspath, 0770, 0740);
install/install.queries.php:1118:                            $result = chmodRecursive($session_abspath.'/files', 0770, 0770);
install/install.queries.php:1121:                            $result = chmodRecursive($session_abspath.'/upload', 0770, 0770);
sources/core.php:116:                chmodRecursive($SETTINGS['cpassman_dir'].'/install', 0755, 0440);

#
# ls -al install/
total 344
drwxrwxr-x.  6 apache apache  4096 Aug 22 01:47 .
drwxrwxr-x. 10 apache apache  4096 Aug 22 01:47 ..
drwxrwxr-x.  3 apache apache    41 Aug 22 01:47 css
drwxrwxr-x.  2 apache apache   196 Aug 22 01:47 images
-rw-rw-r--.  1 apache apache  1181 Aug 22 01:47 install.css
-rw-rw-r--.  1 apache apache 12348 Aug 22 01:47 install.js
-rw-rw-r--.  1 apache apache 14689 Aug 22 01:47 install.php
-rw-rw-r--.  1 apache apache 66288 Aug 22 01:47 install.queries.php
drwxrwxr-x.  3 apache apache    82 Aug 22 01:47 js
drwxrwxr-x.  2 apache apache    24 Aug 22 01:47 libs
-rw-rw-r--.  1 apache apache 45163 Aug 22 01:47 upgrade_ajax.php
-rw-rw-r--.  1 apache apache  6701 Aug 22 01:47 upgrade_db_1.08.php
-rw-rw-r--.  1 apache apache  3402 Aug 22 01:47 upgrade.js
-rw-rw-r--.  1 apache apache 35208 Aug 22 01:47 upgrade.php
-rw-rw-r--.  1 apache apache 12881 Aug 22 01:47 upgrade_run_2.1.26.php
-rw-rw-r--.  1 apache apache 23718 Aug 22 01:47 upgrade_run_2.1.27.php
-rw-rw-r--.  1 apache apache 36784 Aug 22 01:47 upgrade_run_db_original.php
-rw-rw-r--.  1 apache apache  3448 Aug 22 01:47 upgrade_run_defuse_for_categories.php
-rw-rw-r--.  1 apache apache  3649 Aug 22 01:47 upgrade_run_defuse_for_custfields.php
-rw-rw-r--.  1 apache apache  6541 Aug 22 01:47 upgrade_run_defuse_for_files.php
-rw-rw-r--.  1 apache apache  6702 Aug 22 01:47 upgrade_run_defuse_for_files_step2.php
-rw-rw-r--.  1 apache apache  3653 Aug 22 01:47 upgrade_run_defuse_for_logs.php
-rw-rw-r--.  1 apache apache  3448 Aug 22 01:47 upgrade_run_defuse_for_pwds.php
-rw-rw-r--.  1 apache apache  9613 Aug 22 01:47 upgrade_run_encryption_pwd.php
-rw-rw-r--.  1 apache apache  3083 Aug 22 01:47 upgrade_run_encryption_suggestions.php
-rw-rw-r--.  1 apache apache  7160 Aug 22 01:47 upgrade_run_final.php
-rw-rw-r--.  1 apache apache  1619 Aug 22 01:47 upgrade_scripts_manager.php
#
```

**Comment by nilsteampassnet** [comment]
This function was not declared in the scope of this usage.
I have fixed this.

Thank you for reporting with all those details.

**Comment by wirerydr** [comment]
Tested with a fresh install of development commit 67f1f4a.  Confirmed fixed - the installation process completes (after two tries - see note below), and the main login page is displayed after clicking "_Move to home page_".  Providing correct administrative credentials causes a successful login.

**Note:** When testing this I did encounter a **_Defuse\\\\Crypto\\\\Exception\\\\BadFormatException: Encoded data is shorter than expected_** error.  However repeating the installation attempt (without first resetting the environment) did not reproduce this error.  In any event I believe this is unrelated to the current issue.  If I can get it to reproduce reliably, then I will raise as a separate issue.

Thank you very much for fixing this.  Closing this issue - WireRydr
